agent/engine/dependencygraph/graph_test.go

```diff
@@ -133,28 +133,28 @@ func TestDependenciesAreResolvedWhenSteadyStateIsRunning(t *testing.T) {
 		},
 	}
 
-	err = DependenciesAreResolved(php, task.Containers, "", nil)
+	err = DependenciesAreResolved(php, task.Containers, "", nil, nil)
```

Comment:
nit: do you want to put these tests in a table? 😁

Reply:
It looks like these tests have a single task and container dependencies, and the containers' known statuses are changed each time and checked if the dependencies are resolved. I don't think this can be put into a table since there is no pattern associated.